src/index.ts

```diff
@@ -16,7 +16,7 @@ export type Options = {
   exclude?: FilterPattern
   watch?: boolean
   minify?: boolean
-  target?: Target
+  target?: string | string[]
```

Comment:
This just copied the type that esbuild uses in [main.d.ts](https://unpkg.com/browse/esbuild@0.6.3/lib/main.d.ts#LC9)